backend/src/admin/admin.service.ts

```diff
@@ -31,4 +26,17 @@ export class AdminService {
         details.email = details.email.toLowerCase();
         await this.adminRepo.save(details);
     }
+
+    async registerAdmin(registrationData: RegisterAdminDto): Promise<any> {
```

Comment:
In my opinions this should return something else than any. Maybe the created object ie. Admin or boolean?